test/end-to-end/test_pkg_download.sh

```diff
@@ -7,21 +7,25 @@
 # for cleaner code and more fine grained tests. However, they are a
 # bit of a pain to program in bash. This is intended to provide
 # minimal testing pending our figuring out the best approach for
-# command line testing. 
+# command line testing.
 #
+# Assumptions:
+# 1. ACCEPTANCE_HAB_AUTH_TOKEN or HAB_AUTH_TOKEN Environment variables are set and valid
+# 2. ${CACHE_DIR} can be set to a writable location on the filesystem
 
 set -euo pipefail
 
 # Test the Habitat package downloader.
 #
 # Uses the `HAB_INTERNAL_BLDR_CHANNEL` environment variable to control
 # the base packages channel for the exporter.
-HAB=$HAB_TEST_CMD || "hab"
+HAB=${HAB_TEST_CMD:-hab}
 CACHE_DIR="test-cache"
 IDENT_FILE="ident_file"
+HAB_AUTH_TOKEN=${ACCEPTANCE_HAB_AUTH_TOKEN:-${HAB_AUTH_TOKEN}}
```

Comment:
This doesn't appear to be needed (and also seems to fail if the fallback `HAB_AUTH_TOKEN` isn't present)

Reply:
I can remove that line. From what I can see, this will mean `HAB_AUTH_TOKEN` must be set somehow in the pipeline. 

I was thinking it would be made available from `ACCEPTANCE_HAB_AUTH_TOKEN` via expeditor here: https://github.com/habitat-sh/habitat/pull/7104/files#diff-9f269bb242eb760dee635951a1a1226eR382

Reply:
I agree, I don't see this as necessary. I've removed it in this [commit](https://github.com/habitat-sh/habitat/pull/7104/commits/6c8b27a8f8dcb2c3b80b31f1ee7dbe19ce898d25)